### PR TITLE
Fields EditTemplate: Correct positioning of dropdown

### PR DESCRIPTION
**Pull request by BurningTreeC**

### CODE REVIEW

File: core/ui/EditTemplate/fields.tid [hunk 1]
```diff
@@ -71,6 +71,7 @@ $value={{{ [<newFieldValueTiddler>get[text]] }}}/>
 <em class="tc-edit">
 <<lingo Fields/Add/Prompt>>&nbsp;&nbsp;
 </em>
+<div class="tc-edit-field-add-name-wrapper">
 <span class="tc-edit-field-add-name">
```
Comment (pmario) [inline core/ui/EditTemplate/fields.tid:75]: is the span obsolet after adding a wrapper? imo it should be

Comment (BurningTreeC) [reply core/ui/EditTemplate/fields.tid]: thanks @pmario , yes it is, I've removed it

**Comment by Jermolene** [comment]
Hi @BurningTreeC apologies there is another conflict here

**Comment by BurningTreeC** [comment]
Resolved @Jermolene

**Comment by Jermolene** [comment]
Thanks @BurningTreeC

**Pull request merged**